### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by bettybao1209**
add readme to help use the plugin

### CODE REVIEW

**Comment by bettybao1209** [comment]
@meevee98 hi, I wrote a readme giving the description about the commands of the plugin, as well as some notable points to ease the use. Hope to have your opinion.

File: src/PerformanceMonitor/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+It is a plugin to create performance metrics used to benchmark MEO3. You can follow the instructions on [neo-modules](https://github.com/neo-project/neo-modules/blob/master/README.md) to install the plugin.
```
Comment (melanke) [inline src/PerformanceMonitor/README.md]: I think you meant "Neo3" instead of "Meo3"

Comment (bettybao1209) [reply src/PerformanceMonitor/README.md]: Yes, sorry for the typo.

File: src/PerformanceMonitor/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+It is a plugin to create performance metrics used to benchmark NEO3. You can follow the instructions on [neo-modules](https://github.com/neo-project/neo-modules/blob/master/README.md) to install the plugin.
+
+>Note: 
+> 1. Add the `System.Diagnostics.PerformanceCounter` as a dependency in neo-cli to use the `check disk` command.
+> 2. Install `RpcClient` module to use the `rpc time` command.
```
Comment (meevee98) [inline src/PerformanceMonitor/README.md]: Maybe also include a note here informing that it's required to start consensus before using `confirmation time` and `payload time` commands

Comment (bettybao1209) [reply src/PerformanceMonitor/README.md]: I add the note in the Consensus Algorithm part.

**Pull request merged**